force-app/main/default/objects/Journey_Task__c/fields/Estimated_Completion_Date__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Estimated_Completion_Date__c</fullName>
+    <externalId>false</externalId>
+    <formula>Journey__r.Onboarding_Start_Date__c + Template_Task__r.Relative_Days_to_Complete__c</formula>
```

Comment:
You can't rely on the days to complete on the template task, as this may be changed for future journeys. You need to store the relative days to complete on the journey task itself.